firedrake/interpolation.py

```diff
@@ -304,6 +368,88 @@ def _interpolator(V, tensor, expr, subset, arguments, access):
         return copyin + (parloop, ) + copyout
 
 
+@singledispatch
+def rebuild(element, expr):
+    raise ValueError("Not expecting %s" % element)
+
+
+@rebuild.register(finat.DiscontinuousLagrange)
+def rebuild_dg(element, expr):
+    # Create a quadrature element for a runtime point with weight 1
+    # to represent the runtime tabulated point, on the expression's
+    # mesh reference cell, onto which we interpolate
+    if element.degree != 0:
+        raise ValueError("Can only runtime interpolate onto DG0")
+    expr_tdim = expr.ufl_domain().topological_dimension()
+    # gem.Variable name starting with rt_ forces TSFC runtime tabulation
+    runtime_point = gem.Variable('rt_X', (expr_tdim,))
```

Comment:
Make `UnknownPointSet` and `runtime_points = gem.Variable('rt_X', (expr_tdim, num_points))`